pgwatch2/metrics/stat_statements/9.2/metric.sql

```diff
@@ -70,7 +70,7 @@ FROM (
             total_time DESC
         LIMIT 100) a
 UNION
-SELECT
+select /* pgwatch2_generated */
```

Comment:
Not a bug bat rather code style. Uppercase `SELECT` was replaced with lower cases but all other keywords are still uppercased. I'm for uppercase keywords personally.

Reply:
`/* pgwatch2_generated */  is indeed just to not exclude monitoring-generated stuff from dashboards etc. But agreed, should adhere to a unified style...uppercase probably indeed.

Reply:
Btw Pavlo, do you maybe have an idea how to automatically rewrite all those SQL queries - just good old `grep -i` or there's something better?